README.md

```diff
@@ -109,3 +113,8 @@ ConfusionFlow is currently maintained by [Peter Ruch](https://github.com/gfrogat
 
 ## Feedback
 We would be really grateful for any [feedback](https://github.com/confusionflow/confusionflow/issues/new?template=feedback.md) via the repository's issues section.
+
+## Getting Involved
+- You can ask questions on our mailing list [confusionflow-dev@googlegroups.com](https://groups.google.com/forum/#!forum/confusionflow-dev).
+- Please report bugs by submitting a [GitHub issue](https://github.com/ConfusionFlow/confusionflow/issues/new?template=bug_report.md).
+- As we currently don't have the resources to support review of external contributions, we aren't accepting any external pull requests right now.
```

Comment:
I don't expect so many contributions. Hence, I would remove this sentence. It also sounds a bit harsh and is contracticting of the idea of going open source.

Reply:
I somewhat agree and removed it for now. Let's discuss this next week.